test/common/TestReverseRoutesOnly.scala

```diff
@@ -0,0 +1,7 @@
+package rulesplayroutes.test.common
```

Comment:
How come you put this in `common`?

Reply:
I don't think this is necessary, but I don't want to block the merge on this. I'll ask you about it on Monday.